integration-tests/reactive-messaging-kafka/src/main/resources/application.properties

```
@@ -0,0 +1,13 @@
+quarkus.log.category.kafka.level=WARN
+quarkus.log.category.\"org.apache.kafka\".level=WARN
+quarkus.log.category.\"org.apache.zookeeper\".level=WARN
+
+# enable health check
+quarkus.kafka.health.enabled=true
+kafka.bootstrap.servers=localhost:19092
+
+mp.messaging.outgoing.people-out.connector=smallrye-kafka
+mp.messaging.outgoing.people-out.topic=people
+mp.messaging.outgoing.people-out.value.serializer=io.quarkus.kafka.client.serialization.JsonbSerializer
+mp.messaging.incoming.people-in.connector=smallrye-kafka
+mp.messaging.incoming.people-in.value.deserializer=io.quarkus.it.kafka.PersonDeserializer
```

I think the incoming sink is missing the topic.
```suggestion
mp.messaging.incoming.people-in.connector=smallrye-kafka
mp.messaging.incoming.people-in.value.deserializer=io.quarkus.it.kafka.PersonDeserializer
mp.messaging.incoming.people-in.topic=people
```

---

Added the topic for the incoming sink. I had used as a guide the example project (https://quarkus.io/guides/kafka#configuring-the-kafka-connector) which does not actually set the topic for the incoming sink.